packages/swarm-styles/lib/spacing.css

```diff
@@ -1,3 +1,17 @@
 :root {
-    --space: 16px;
-}
\ No newline at end of file
+    --space: var(--size-space-default);
```

Comment:
@hsbacot 

- Does this css file look correct to you now?
- Is there any other CSS I should be adding?
- Is `--responsive-space` the correct variable to assign here? Or should it just be `--space` for everything?

Reply:
@sadafie we need to have all of the definitions for space sizes here when they are responsive.

It should be `--space` for everything. The spacing should adjust based on viewport size.